layers/+lang/scheme/README.org

```diff
@@ -183,15 +184,40 @@ When enabled the symbol =🆂= should be displayed in the mode-line.
 | ~SPC m e l~ | Evaluate line             |
 | ~SPC m e r~ | Evaluate region           |
 
-** REPL
-
-| Key binding | Description                 |
-|-------------+-----------------------------|
-| ~g j~       | geiser-repl-next-prompt     |
-| ~g k~       | geiser-repl-previous-prompt |
-| ~SPC m h~   | geiser-doc-symbol-at-point  |
-| ~SPC m i~   | geiser-repl-interrupt       |
-| ~SPC m c~   | geiser-repl-clear-buffer    |
-| ~SPC m m~   | geiser-repl-import-module   |
-| ~SPC m u~   | geiser-repl-unload-function |
-| ~SPC m q~   | geiser-repl-exit            |
+** REPL-mode
+
+*Insert state*
+| Key binding | Description           |
+|-------------+-----------------------|
+| ~S-RET~     | Insert newline indent |
+| ~C-l~       | Clear buffer          |
+| ~C-d~       | Exit                  |
+
+*Normal state*
+| Key binding | Description                                       |
+|-------------+---------------------------------------------------|
+| ~g j~       | Goto next prompt                                  |
```

Comment:
you may combine them
|~g j~ / ~]]~ | Go to next prompt     |
|~g k~ / ~[[~ | Go to previous prompt |